package.json

```diff
@@ -1,7 +1,7 @@
 {
   "homepage": "https://dinofrontend.github.io/dino_ui_library",
   "name": "dino_ui_tools",
-  "version": "2.2.8",
+  "version": "2.2.9",
```

Comment:
you dont need to update version.

feat: add new login method: This would normally increment the MINOR version.
fix: correct login redirect issue: This would normally increment the PATCH version.
If the current version was 1.2.3, the new version after this commit would be 1.3.0 (since the feat causes a MINOR version bump). 

master mergeluc commit message-y es structureov kdnes, versiony update klini, dzerov update anel el petq chi